src/features/shared/markdown/LinkInterceptor.tsx

```diff
@@ -2,13 +2,17 @@ import { LinkHTMLAttributes, MouseEvent, useCallback } from "react";
 import { useAppSelector } from "../../../store";
 import { useIonRouter } from "@ionic/react";
 import { useBuildGeneralBrowseLink } from "../../../helpers/routes";
+import styled from "@emotion/styled";
 
 const COMMUNITY_RELATIVE_URL =
   /^\/c\/([a-zA-Z0-9._%+-]+(@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,})?)\/?$/;
 
-export default function LinkInterceptor(
-  props: LinkHTMLAttributes<HTMLAnchorElement>
-) {
+const LinkInterceptor = styled(LinkInterceptorUnstyled)`
+  -webkit-touch-callout: default;
+  -webkit-tap-highlight-color: transparent;
```

Comment:
Thanks for the PR! :)

At least on my device (iPhone 13 mini) this doesn't seem to do anything